src/ui/Auth.tsx

```diff
@@ -1,22 +1,23 @@
 import { authenticateUser } from '../data'
 import googleLogo from '../icons/google-logo.png'
 import githubLogo from '../icons/github-logo.png'
+import { Provider } from '@supabase/supabase-js'
 
 const providers = [
   {
-    provider: 'google',
+    provider: 'google' as Provider,
     label: 'Sign in with Google',
     logo: googleLogo,
   },
   {
-    provider: 'github',
+    provider: 'github' as Provider,
     label: 'Sign in with Github',
     logo: githubLogo,
   },
 ]
```

Comment:
```suggestion
] as const
```
will fix your type issue.

Reply:
Not exactly it will force the provider property to be "any" whereby it actually is a valid Provider type.

Reply:
It will make it a string literal.